projects/control-service/projects/pipelines_control_service/src/main/resources/k8s-data-job-template.yaml

```diff
@@ -1,7 +1,7 @@
 # Copyright 2021 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
-apiVersion: batch/v1beta1
+apiVersion: batch/v1
 kind: CronJob
```

Comment:
In case we want to switch the VDK defaults to latest, maybe we need to reconsider releasing a major version, and notifying in advance the users so they have time to switch the configurations accordingly. I don't see however why such a change is needed, since it is still configurable, haven't heard a use-case that requires this yet

Reply:
Yeah maybe this needs to be a major release. 
Its not switching the defaults. 
V1 is the only supported value now. v1Beta was deprecated years ago and is being completed removed from the apis now.

Reply:
I am not familiar with any use case that requires such a change, why is it better t explicitly purposefully remove backwards compatibility?

Reply:
Because it is really really old and kubernetes are not supporting it anymore. 
It was a beta version of v1 while they tested stuff out. 
But really everyone should have moved to v1 when it became available because that is the stable version.

Reply:
So in fact this is not quite "migrating" to newest k8s version, because currently VDK is able to run on latest version, right? We ar dropping the older versions support, and my suggestion is to rename the PR accordingly.

Reply:
I believe this PR could be split in two. One is the template apiVersion defaults, set to v1 that is required for running on Kubernetes latest.